### ISSUE TITLE
Trouble with setting up Malmo

### ISSUE THREAD

**Issue by reu24** [rahul-raj/Java-Deep-Learning-Cookbook#33]
**Describe the bug**
I've tried to install Malmo on my Windows PC, like the book descriped but when I ran the example programms from Malmo they did nothing and if I ran the code from the book I've got a long error log.
The error log: https://textuploader.com/16dsi

**Expected behavior**
It should run the Malmo example programm.

**Comment by rahul-raj** [comment]
@reu24 What are your observations on debugging? 
Have you googled it?  As per the error stack, it seems like minecraft specific issue, not an issue with the code.

**Comment by reu24** [comment]
Thank you for the fast answer.
I've already googled the error, but I found nothing that made it work.
The error occures at `dql.train();` and I could debug it until `agent_host.startMission(mission, clientPool, missionRecord, 0, "rl4j_0");` in MalmoEnv. 
I think that something is wrong with the installation. But I followed step by step the guidance and also tried it again. The only thing I couldn't follow was when I had to execute malmo_install.ps1 since it gave me an error. But I installed the dependencies myself.
